[IMP] crm: update win loss report documentation for Odoo 17

MaxKeilOdoo opened the pull request:
Updated the "Check Your Win/Loss Ratio" documentation for 17.0.

-Changed title to "Create a Win/Loss Report"
-Added more context to the intro about what a Win/Loss Ratio is
-Revised filter and group by selections because "Total Revenue" option from 16.0 is no longer present
-Added optional filters and group by options that could improve the usefulness of the report for certain users (like sales managers)
-Added a new section for "Additional Measurements and Views"
-Added a new section for how to "Add a Win/Loss Report to Dashboards", so users do not need to pull the report over and over
-Updated images throughout
-Updated language throughout, removed instances of "you" and future tense

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/6746).

MaxKeilOdoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,85 @@
+#. Filters: *Won*.
+#. Filters: *Lost*.
+
+   - Optional: Removing or adjusting the *Created On* filter to reflect the last 30 days, the last
```
Change "Removing or adjusting" to "Remove or adjust"
```suggestion
   - Optional: Remove or adjust the *Created On* filter to reflect the last 30 days, the last
```

Felicious commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,85 @@
+   :align: center
+   :alt: Win/Loss report in pivot view showing the Measures dropdown menu.
+
+Select *Blank Spreadsheet* and click *Confirm*. Name the new spreadsheet,
+and add charts or additional formulas as desired.
```
Have you ever tried the command `make review`? it helps catch instances where you can fit a few more words onto the line and get it as close to 100 characters as possible. 

To run make review, go to github and copy the file path, in this case
`content/applications/sales/crm/performance/win_loss.rst`
then, REMOVE content/, so u are left with
`applications/sales/crm/performance/win_loss.rst`

Go to terminal, and in the documentation folder, run `make review`
and then, enter the path we just made
When they ask the question about 100 char, just press the Enter button on your keyboard
![image](https://github.com/odoo/documentation/assets/23611711/8d760867-d397-4bef-a5e9-a53166d1c878)

Then, you'll see messages like this:
![image](https://github.com/odoo/documentation/assets/23611711/cb8c36c0-2ed3-45c9-ade9-205872db4fa3)

Saying that in lines 36, 45, 51, 63, 70, and 73, u have an early line break and u can reformat. (For example, shove more words from line 71 onto 70) 
```suggestion
Select *Blank Spreadsheet* and click *Confirm*. Name the new spreadsheet, and add charts or
additional formulas as desired.
```

Felicious commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,85 @@
-=========================
-Check your Win/Loss Ratio
-=========================
+========================
```
Thanks for changing the branch name to CRM! Reminder to change the name of the PR to 
[IMP] CRM: ... too, but clicking the edit button at the top of the PR 
![image](https://github.com/odoo/documentation/assets/23611711/02ffa5ce-dd64-4ad9-ae7e-ed66a4208af6)

Felicious commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,85 @@
+   :align: center
+   :alt: Win/Loss report in a spreadsheet showing the Add to Dashboard option.
+
+The Win/Loss report is now viewable in the *Dashboards* app. This view automatically updates to
```
great job remembering  that app names are italicized!

Felicious reviewed (changes requested):
Great job with your first PR, @MaxKeilOdoo ! Very proud that you were able to push a PR up on your own, and you followed the instructions correctly to tag the relevant reviewers. I like that your PR description is so detailed too, it'll help us in the future keep track of changes/updates to the docs (:

Overall, I just had some formatting reminders and suggestions to keep your document consistent with the style the rest of the team writes in. Namely, we use `guilabel`s for website elements in Odoo that we can *see*, and `menuselection` when telling the reader where to click and go.

Don't worry, you'll get used to this and it'll become second nature the more documentation you write! :smiley: 

Let me know when this doc is ready for a second look, and feel free to reach out anytime for clarification on any of my comments, or you're having some RST issues. Always happy to help! 

Most importantly, you're doing great, Max! RST is *basically* a coding language, since you're writing it in a specific way to build a webpage from the bottom-up! :flushed: Isn't that cool? :smile:

MaxKeilOdoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,85 @@
+Click the chart icons along the top to switch the graph type from a Bar Chart to a Line Chart or Pie
+Chart.
+
+Additionally, the report can be changed from a Graph view to a Pivot view, Cohort view, or
```
So for this one, you're suggesting I italicize the first instance of "Bar Chart... Line Chart... Pie Chart", and then leave the second instances as they are?

Felicious commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,100 @@
+By default, the win/loss report measures the total *Count* of leads as a *Bar Chart*, but there are
+many options for measures and views.
+
+Go to :menuselection:`CRM --> Reporting --> Pipeline`. Select any desired filters and group by
```
In line 9, in the navigation to CRM, there's an "app" that follows the app name
> :menuselection:`CRM app --> Reporting --> Pipeline`

I think we should keep this navigational instruction (:menuselection:`[app name] --> [heading] --> [click the option]`) consistent throughout the document. So, we could either remove the word "app" from line 9, or add it in here.

```suggestion
Go to :menuselection:`CRM app --> Reporting --> Pipeline`. Select any desired filters and group by
```
Friendly reminder that there's another navigational instruction like this one on line 72!

MaxKeilOdoo replied:
Good catch! Thank you. I'll fix the one on line 72 as well.

Felicious reviewed (approved):
Great job with this doc, @MaxKeilOdoo! It's looking really good and the document passed all of the C.I. checks! So great to see you getting used to all of the githubby tools, so quickly!

I just had a few minor comments, but after you address them, this doc is ready for the next stage of review :blush: 

I'll help you with  squashing the extra commits tomorrow in person, or you can ask hojo (:

MaxKeilOdoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,100 @@
+   :align: center
+   :alt: View of CRM page showing the Search Panel open and the Won and Lost filters highlighted.
+
+After making the above selections, the :guilabel:`Search Panel` should read:
+:guilabel:`Created On: YYYY`, :guilabel:`Won or Lost`, :guilabel:`Salesperson > Active`. If so, the
+graph now displays the *Count* of wins and losses for each salesperson. Hover over the chart to see
+the percentage of wins (true) or losses (false). Click on a section of the chart to open a list of
+those accounts.
```
I did not realize you could break up the text after :guilabel: but I guess that makes sense as long as you have the backticks.

MaxKeilOdoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,100 @@
+dashboard for easier viewing in the future.
+
+Go to :menuselection:`CRM --> Reporting --> Pipeline`. Select any desired filters and group by
+options for the report. Now, click on the :guilabel:`Pivot view` button to the right of the Search
```
I like both of these changes. Agree that we should only use guilabel with the actual text that shows up when you hover over the button/icon.

MaxKeilOdoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,100 @@
+   :alt: View of CRM page showing the Search Panel open and the Won and Lost filters highlighted.
+
+After making the above selections, the :guilabel:`Search Panel` should read:
+:guilabel:`Created On: YYYY`, :guilabel:`Won or Lost`, :guilabel:`Salesperson > Active`. If so, the
```
I don't know if we need to change this one.

When I open CRM in 17.0, I land at the Kanban view. I then open the Reporting menu and click Pipeline. That lands me at the Pipeline Analysis page that already has a bar graph open.

When I open the search bar dropdown and select the filters/group bys, that automatically changes the bar graph to reflect those selections. So really, all they need to do is click off the search bar dropdown (to make it disappear) and then they are viewing the bar graph by default.

I callout the chart icons later because I think they're worth knowing about as additional views, but they aren't relevant for just pulling the base report. Thoughts?

jero-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,101 @@
+Additional measurements and views
+=================================
+
+By default, the win/loss report measures the total *Count* of leads as a Bar Chart, but there are
```
```suggestion
By default, the win/loss report measures the total :guilabel:`Count` of leads as a Bar Chart, but there are
```
I am erring on the side of caution adding a guilabel here, I will defer to Sam but I think since it refers to an interactive element on the screen it should probably have one? (I am happy to be corrected 😄)

jero-odoo reviewed (approved):
Hi @MaxKeilOdoo , great work! I included a few formatting changes, but the rest are suggestions. 
One other thing- there is a document specifically for Reporting (https://www.odoo.com/documentation/17.0/applications/general/reporting.html) that has some good information about using the various views/filters/measures. It may be worth it to add a "See Also" block at the end of this doc for some additional information.
Let me know if you want to discuss anything, I'd be happy to go over any of them with you! Great work 👍

MaxKeilOdoo commented:
Hi @odoo/us-doc-review 

This is ready for final review. Thanks!

StraubCreative reviewed (changes requested):
Hi @MaxKeilOdoo 

Good rewrite so far 🙂 

I'd like to expand the content considerably more— for line 16 and beyond, it seems that we're covering a *very* specific use case in a concise fashion, however I'd like to see the core parameters of the win/loss report identified and described first, in clear detail, and then we can branch out to additional parameters and advanced use cases later in the doc.

The utility of this doc is so that a reader can come here, and walk away with a complete understanding of: 
- what makes up a basic vs. advanced win/loss report
- what their options are for **Filters** and **Group by** and when/how to use each to achieve specific outcomes
- how to further manipulate or export win/loss data using **Measures** menu and **Dashboards** app

As well I think we're missing a filter here— in order to get an accurate read on win/loss (or anything to do with leads) we need to have an ANY statement: `Active is set` OR `Active is not set` so that lost leads are actually included in the report. This can also be accomplished by selecting both **Active** and **Inactive** in the **Filters** part of the mega menu.

MaxKeilOdoo commented:
Hi @ksc-odoo , I've expanded the content of the win/loss documentation and would love it if you could double-check that I got everything correct. Thank you!

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,249 @@
+
+After creating a basic win/loss report in the *CRM app's Pipeline Analysis*, consider
+:ref:`selecting Pivot View <reporting/views>` to see the data in a table format. By default,
+:guilabel:`Pivot View` groups basic win/loss reports by :guilabel:`Stage` and measures
```
since the words 'Pivot View' are immediately seen over the button for it, there's no need to use a GUI in this instance. Same goes for 'Stage' and 'Count' -- but 'Expected Revenue' *DOES* need a GUI because you can see it on the UI/it's present on the screen

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,249 @@
+   :align: center
+   :alt: The same win/loss report displays different information when using Pivot View or List View.
+
+Win/Loss in pivot view
+======================
```
This section may be better served above, if you take on the approach suggested -- wherein you break down every element of the Pipeline Analysis page (including all the different view options)

ksc-odoo replied:
And the following portion about inserting it into a spreadsheet would be a great addition, too

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,249 @@
+Win/Loss in pivot view
+======================
+
+After creating a basic win/loss report in the *CRM app's Pipeline Analysis*, consider
```
*CRM* app's :guilabel:`Pipeline Analysis` page...

In this instance, the words 'Pipeline Analysis' appear on the U/I...before, it was mentioned in the intro -- but at that point, the reader was just being introduced to the concept, and wasn't "officially" looking at a page with that title -- hence the use of italics was deemed okay.

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,249 @@
+Advanced filters and groupings
+------------------------------
+
+The *Pipeline Analysis* can be customized with various filters and group by options. After
```
At this point, the reader is expected to be on the Pipeline Analysis page, so the use of GUI here is required -- because that title is actually on the screen

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,249 @@
+------------------------------
+
+The *Pipeline Analysis* can be customized with various filters and group by options. After
+creating a basic win/loss report, click on the :guilabel:`Search bar` and consider adding advanced
```
I believe they would be clicking on a down arrow icon beside the search bar to reveal the drop-down menu of options.

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,249 @@
+
+The *Pipeline Analysis* can be customized with various filters and group by options. After
+creating a basic win/loss report, click on the :guilabel:`Search bar` and consider adding advanced
+filters and groupings to customize the report for different purposes. Some useful options
+include:
+
+- **Created On**: Adjusting this filter to a different period of time such as the last 30 days
+  or the last quarter can provide more timely results.
+- **Add Custom Filter**: Clicking :guilabel:`Add Custom Filter` and scrolling through the numerous
+  options in the drop-down opens up additional search criteria like :guilabel:`Last Stage Update` or
+  :guilabel:`Lost Reason`.
+- **Multiple Groupings**: Adding multiple *Group By* selections can split results into more
+  relevant and manageable chunks.
+
+  - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of leads in
+    each :guilabel:`Stage` by who they were assigned to.
+  - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate more
+    sales.
+- **Add Custom Group > Active**: Clicking :menuselection:`Add Custom Group --> Active` separates the
+  results in each stage into *Won (true)* or *Lost (false)*. This can show at what stage leads are
+  won and lost.
```
I'd suggest expanding this portion to touch on every single option available. Our doc readers prefer over-explanation. You can do this in a simple list format.

Again, I think this portion should appear above, where you introduce the entire page to the reader...

But I'd probably go with something like...

Filters
-------

The :guilabel:`Filters` section allows users to add custom filters to the results on the
:guilabel:`Pipeline Analysis` page.

- :guilabel:`My Pipeline`: only shows personal leads/opportunities.
- :guilabel:`Opportunities`: only shows opportunities.
- :guilabel:`Leads`: only shows leads.

...and so on...

and then you can start another section for 'Group By' and then 'Favorites'

doing so makes things super-organized and easy to reference

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,249 @@
+Additional measurements and views
+---------------------------------
+
+In addition to filters and group by options, the *Pipeline Analysis* can also display different
+measurements, chart types, and :ref:`report views <reporting/views>`. Need to know something other
+than the total count of leads? Change the measurement to something more valuable. Want to see the
+report as a spreadsheet? Switch to the :guilabel:`Pivot View`.
```
I appreciate the personality and personability of this type of phrasing, but documentation is purely meant to instruct and guide the reader along the path towards usability. We don't really need to "suggest" or encourage them to "try" different views/elements. We're kinda just meant to be as sterile as possible with documentation - that way, they can quickly get the information they need.

in essence docs are...

Title of thing you want to know about
- brief explanation of thing you want to know about, and how Odoo does it.

Next heading of section to show how to set up for thing you want to know about
- here's how to set up Odoo to do the thing you want to know about

Page of thing you want to know about
- here's how to get to the page where you can do the thing you want to know about. 
- here's what everything means on the page where you can do the thing you want to know about.
- here are all the various view and filter options on the page where you can do the thing you want to know about.

...and so on...

It's just an in-depth, super informative, step-by-step instruction manual for the reader. It's our job to make sure we've left no stone unturned with our explanations. Doing so ensures there's little to no confusion, and they can feel adequately taught throughout their documentation experience

ksc-odoo reviewed (commented):
Hey @MaxKeilOdoo ! 👋 

First, I have to commend you on a truly valiant effort put forth on this doc -- your first of many to come! 👍 

You're really taking to the RST formatting quite well -- and don't worry about making mistakes along the way. We ALL did (and continue to) 😆 , so just keep "rolling with the punches" and everything will become second-nature for you in no time!

I finished reviewing this doc, and left you as many detailed notes as I could along the way. You'll notice that, at around the 150-ish line mark, the comments ceased. That's only because I think you'll want to reformat some things, and apply some different tactics to this doc during your next attempt at it.

I think, if you apply those suggested methodologies, and work in some of those suggested techniques, it's going to make this doc super informative, well-rounded, and chock full of amazing Odoo knowledge!

That said, I didn't want to give you too many things to consider during your next attempt. No sense in feeling bombarded by suggestions, ya know? So, feel free to digest this info, try your best to implement those adjustments, and tag me again for another look/review whenever you're ready.

I'm also happy to answer any questions you may have, or provide any clarification you may need.

You're already catching on super-fast! Just take things one at a time, try to be as thorough as possible, and try to utilize my "explain it to your dad" approach...once you do that, this PR is gonna be flying through the pipeline.

Once again, though -- FANTASTIC attempt! Looking forward to giving it another review soon! 👍

MaxKeilOdoo commented:
Hi @ksc-odoo please review this once again, I've added multiple new sections and renamed the document "Pipeline Analysis" because it now has more to do with that than just win/loss reporting.

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,249 @@
+.. math::
+   \begin{equation}
+   Win/Loss Ratio = \frac{Opportunities Won}{Opportunities Lost}
+   \end{equation}
```
Is it necessary? No...
But is it math? YES 🔥 

My vote is we keep it 😁 
[!becauseMath](https://media.giphy.com/media/l2YWF00ZX8wOs0p0s/giphy.gif)

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+         :align: center
+         :alt: More details can be added to the list by clicking the Filters icon in the top-right.
+
+      Clicking the gear-shaped :guilabel:`Actions` icon opens a drop-down menu with options for the
+      following:
```
```suggestion
      Clicking the :guilabel:`⚙️ (gear)` icon opens the Actions drop-down menu, with options for the
      following:
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+      Clicking the gear-shaped :guilabel:`Actions` icon opens a drop-down menu with options for the
+      following:
+
+      - :guilabel:`Import records`: Opens a page for uploading a spreadsheet of data, as well as a
+        template spreadsheet for easily formatting that data.
+      - :guilabel:`Export All`: Downloads the list as an xlsx file for Excel.
+      - :guilabel:`Knowledge`: Inserts a view of or link to the list in an article in the
+        *Knowledge* app.
+      - :guilabel:`Dashboard`: Adds the list to :guilabel:`My Dashboard` in the *Dashboards* app.
+      - :guilabel:`Spreadsheet`: Links to or inserts the list in a spreadsheet in the *Documents*
+        app.
```
```suggestion
      - :guilabel:`Import records`: opens a page for uploading a spreadsheet of data, as well as a
        template spreadsheet to easily format that data.
      - :guilabel:`Export All`: downloads the list as an xlsx file for Excel.
      - :guilabel:`Knowledge`: inserts a view of, or link to, the list in an article in the
        *Knowledge* app.
      - :guilabel:`Dashboard`: adds the list to *My Dashboard* in the *Dashboards* app.
      - :guilabel:`Spreadsheet`: links to, or inserts, the list in a spreadsheet in the *Documents*
        app.

```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+        app.
+
+      .. note::
+         At the List View, clicking :guilabel:`New` closes the list and opens the *New Quotation*
+         page. Clicking :guilabel:`Generate Leads` opens a pop-up for lead generation. Neither
+         feature is intended to manipulate the List View.
```
```suggestion
         On the list View, clicking :guilabel:`New` closes the list, and opens the *New Quotation*
         page. Clicking :guilabel:`Generate Leads` opens a pop-up window for lead generation.
         Neither feature is intended to manipulate the list View.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+
+.. _win_loss/reports:
+
+Creating reports
+================
```
```suggestion
Create reports
==============
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+Creating reports
+================
+
+After understanding how to :ref:`navigate the analysis page <win_loss/pipeline>`, the
+:guilabel:`Pipeline Analysis` can be used to create and share different reports. Between the
+pre-made options and custom filter/groupings, almost any combination is possible.
```
```suggestion
After understanding how to :ref:`navigate the pipeline analysis page <win_loss/pipeline>`, the
:guilabel:`Pipeline Analysis` page can be used to create and share different reports. Between the
pre-made options and custom filter/groupings, almost any combination is possible.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+Once created, reports can be :ref:`saved to favorites, shared with other users, and added to
+knowledge articles, dashboards, and spreadsheets <win_loss/save_reports>`.
+
+A few common reports that can be created using the :guilabel:`Pipeline Analysis` are detailed below.
```
```suggestion
A few common reports that can be created using the :guilabel:`Pipeline Analysis` page are detailed
below.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+Win/Loss reports
+----------------
+
+Win/Loss is a metric of the leads/opportunities marked Won or Lost over a period of time. By
+calculating "opportunities won" over "opportunities lost," teams can identify what methods and/or
+individuals are effectively converting leads into sales.
```
```suggestion
Win/Loss is a metric of the leads/opportunities marked "Won" or "Lost" over a period of time. By
calculating "opportunities won" over "opportunities lost," teams can identify what methods and/or
individuals are effectively converting leads into sales.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+calculating "opportunities won" over "opportunities lost," teams can identify what methods and/or
+individuals are effectively converting leads into sales.
+
+A win/loss report filters **the leads/opportunities from the past year, whether won or lost, and
+groups the results by their stage in the pipeline**. Creating this report requires a custom filter
+and grouping the results by :guilabel:`Stage`.
```
```suggestion
A win/loss report filters the leads/opportunities from the past year, whether "Won" or "Lost," and
groups the results by their stage in the pipeline. Creating this report requires a custom filter,
and grouping the results by :guilabel:`Stage`.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+Follow the steps below to create a win/loss report:
+
+1. Navigate to :menuselection:`CRM app --> Reporting --> Pipeline`.
+2. At the :guilabel:`Pipeline Analysis` page, click on the :guilabel:`down arrow` next to the search
+   bar to open a menu of filters and groupings.
```
```suggestion
2. On the :guilabel:`Pipeline Analysis` page, click on the :guilabel:`down arrow` icon, next to the
   search bar, to open a drop-down menu of filters and groupings.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+   :guilabel:`Country`.
+6. Clicking that first field reveals a sub-menu with numerous options to choose from. From this
+   sub-menu, locate and select the :guilabel:`Active` option. Doing so, auto-populates the remaining
+   fields. The first field reads: :guilabel:`Active`. The second field reads: :guilabel:`is`. And
+   lastly, the third field reads: :guilabel:`set`. In total, the rule reads: :guilabel:`Active is
+   set`.
```
```suggestion
   fields.
   
   The first field reads: :guilabel:`Active`. The second field reads: :guilabel:`is`. And lastly,
   the third field reads: :guilabel:`set`.
   
   In total, the rule reads: :guilabel:`Active is set`.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+   :align: center
+   :alt: The Add Custom Filter menu showing two rules: (1) Active is set, and (2) Active is not set.
+
+The report now displays the total :guilabel:`Count` of leads, whether won or lost, grouped by their
+stage in the CRM pipeline. Hover over a section of the report to see the number of leads in that
+stage.
```
```suggestion
The report now displays the total :guilabel:`Count` of leads, whether "Won" or "Lost," grouped by
their stage in the CRM pipeline. Hover over a section of the report to see the number of leads in
that stage.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+   :align: center
+   :alt: A basic win/loss report showing all leads whether won or lost grouped by stage.
+
+Customizing win/loss reports
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
```suggestion
Customize win/loss reports
~~~~~~~~~~~~~~~~~~~~~~~~~~
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+
+   .. tab:: Filters and groups
+
+      To add more filters and groups, click the :guilabel:`down arrow` next to the search bar and
+      select one or more options from the drop-down menu.
```
```suggestion
      To add more filters and groups, click the :guilabel:`down arrow` icon, next to the search bar,
      and select one or more options from the drop-down menu.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+
+      Some useful options include:
+
+      - **Created On**: Adjusting this filter to a different period of time such as the last 30 days
+        or the last quarter can provide more timely results.
+      - **Add Custom Filter**: Clicking :guilabel:`Add Custom Filter` and scrolling through the
+        numerous options in the drop-down opens up additional search criteria like :guilabel:`Last
+        Stage Update` or :guilabel:`Lost Reason`.
+      - **Multiple Groupings**: Adding multiple :guilabel:`Group By` selections can split results
+        into more relevant and manageable chunks.
+
+        - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of
+          leads in each :guilabel:`Stage` by who they were assigned to.
+        - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate
+          more sales.
+      - **Add Custom Group > Active**: Clicking :menuselection:`Add Custom Group --> Active`
+        separates the results into Won (:guilabel:`true`) or Lost (:guilabel:`false`). This shows
+        at what stage leads are being marked won or lost.
```
```suggestion
      - :guilabel:`Created on`: adjusting this filter to a different period of time, such as the
        last 30 days, or the last quarter, can provide more timely results.
      - :guilabel:`Add Custom Filter`: clicking this option, and scrolling through the numerous
        options in the drop-down menu, opens up additional search criteria, like :guilabel:`Last
        Stage Update` or :guilabel:`Lost Reason`.
      - :guilabel:`Multiple Groupings`: add multiple :guilabel:`Group By` selections to split
        results into more relevant and manageable chunks.

        - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of
          leads in each :guilabel:`Stage`.
        - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate
          more sales.
      - :guilabel:`Add Custom Group > Active`: Clicking :menuselection:`Add Custom Group --> Active`
        separates the results into "Won" (:guilabel:`true`) or "Lost" (:guilabel:`false`). This
        shows at what stage leads are being marked "Won" or "Lost."
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+      pop-up menu, replace the :guilabel:`Stage` grouping with something like
+      :guilabel:`Salesperson` or :guilabel:`Medium`.
+
+      Then click the :guilabel:`Measures` button and add the :guilabel:`Count` of leads back into
+      the report.
+
+      Other useful measures for :guilabel:`Pivot View` include :guilabel:`Days to Assign` and
+      :guilabel:`Days to Close`.
+
+      .. image:: win_loss/win-loss-pivot-view.png
+         :align: center
+         :alt: A win/loss report in Pivot View displays the data in table form.
+
+      .. important::
+         By default, the :guilabel:`Insert In Spreadsheet` button may be greyed out due to the
+         report containing :guilabel:`duplicate group bys`.
+
+         To fix this, replace the :guilabel:`Stage` grouping in the search bar with another option.
+
+   .. tab:: List View
+
+      In :guilabel:`List View`, a win/loss report displays all leads on a single page.
+
+      To add more columns to the list, click the :guilabel:`Filters` icon in the top-right of the
+      page, and select options from the resulting drop-down menu. Some useful filters include:
+
+      - **Campaign**: Shows the marketing campaign that originated each lead.
+      - **Medium**: Shows the marketing medium (Banner, Direct, Email, Google Adwords, Phone,
+        Website, etc.) that originated each lead.
+      - **Source**: Shows the source of each lead (Newsletter, Lead Recall, Search Engine, etc.).
+
+      To better organize the list, click the :guilabel:`down arrow` next to the search bar, and
+      add more relevant groupings or re-organize the existing ones.
+
+      To re-order the nesting, remove all :guilabel:`Group By` options and re-add them in the
+      desired order.
+
+      .. image:: win_loss/win-loss-list-view.png
+         :align: center
+         :alt: A win/loss report in List View displays all leads in an easy-to-read list.
```
This section seems repetitive, and I feel like all of this was adequately explained above. I'd suggest removing it.

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+
+.. _win_loss/save_reports:
+
+Saving and sharing reports
+==========================
```
```suggestion
Save and share reports
======================
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+does not need to be created again in the future. Saved searches automatically update their results
+every time the report is opened.
+
+Additionally, reports can be shared with others or added to spreadsheets/dashboards for greater
+customization and even easier access.
```
```suggestion
Additionally, reports can be shared with others, or added to spreadsheets/dashboards for greater
customization and easier access.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+
+      To save a report for later:
+
+      At the :guilabel:`Pipeline Analysis` page, click the :guilabel:`down arrow` next to the search
+      bar, and under the :guilabel:`Favorites` heading, click :guilabel:`Save current search`.
```
```suggestion
      On the :guilabel:`Pipeline Analysis` page, click the :guilabel:`down arrow` icon, next to the
      search bar, and under the :guilabel:`Favorites` heading, click :guilabel:`Save current
      search`.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+      At the :guilabel:`Pipeline Analysis` page, click the :guilabel:`down arrow` next to the search
+      bar, and under the :guilabel:`Favorites` heading, click :guilabel:`Save current search`.
+
+      In the drop-down menu that appears, enter a name for the report. By default, the report is
+      named "Pipeline".
```
```suggestion
      In the drop-down menu that appears, enter a name for the report.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+      In the drop-down menu that appears, enter a name for the report. By default, the report is
+      named "Pipeline".
+
+      - Optional: Checking the :guilabel:`Default filter` box sets this report as the default
+        analysis when opening the :guilabel:`Pipeline Analysis` page.
+      - Optional: Checking the :guilabel:`Shared` box makes this report available to other
+        users.
```
```suggestion
      Checking the :guilabel:`Default filter` box sets this report as the default analysis when the
      :guilabel:`Pipeline Analysis` page is opened. Checking the :guilabel:`Shared` box makes this
      report available to other users.
```

ksc-odoo commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+      To add a report in to a spreadsheet:
+
+      - **If in Graph or Pivot View**: Click either the :guilabel:`Insert in spreadsheet` button,
+        and in the pop-up menu that appears, click :guilabel:`Confirm`.
+      - **If in Cohort or List View**: Click the gear-shaped :guilabel:`Actions` icon, hover over
+        :guilabel:`Spreadsheet`, and click either :guilabel:`Insert/Link in spreadsheet`.
+
+        .. image:: win_loss/pivot-view-in-spreadsheet.png
+           :align: center
+           :alt: Pivot View reports especially benefit from being inserted in spreadsheets.
+
+      .. tip::
+         After modifying a spreadsheet and adding additional formulas, consider then adding the
+         entire spreadsheet to a dashboard by clicking :menuselection:`File --> Add to dashboard`.
+
+         In the resulting pop-up menu, name the spreadsheet and select a :guilabel:`Dashboard
+         Section` to house the report. Once complete, click :guilabel:`Create`.
+
+         Using this method, the spreadsheet can be added to a public dashboard instead of only
+         :guilabel:`My Dashboard`.
+
+   .. tab:: Add to a Dashboard
+
+      Adding a report to a dashboard saves it for later and makes it easy to view alongside the rest
+      of :guilabel:`My Dashboard`.
+
+      To add a report to :guilabel:`My dashboard`:
+
+      At the :guilabel:`Pipeline Analysis` page, click the gear-shaped :guilabel:`Actions` icon, and
+      in the resulting drop-down menu, hover over :guilabel:`Dashboard`.
+
+      Under the :guilabel:`Add to my dashboard` heading, enter a name for the report (by default, it
+      is named "Pipeline") and click :guilabel:`Add`.
+
+      To see the saved report, return to the main apps page, and navigate to
+      :menuselection:`Dashboards app --> My Dashboard`.
+
+      .. image:: win_loss/add-to-dashboard.png
+         :align: center
+         :alt: To access the saved report, open the Dashboard app and click My Dashboard.
```
I believe this section can be deleted, as well -- since you've already covered it in great detail previously in the doc.

ksc-odoo reviewed (commented):
@MaxKeilOdoo Just finished reviewing the doc. Great job on expanding on elements and being super thorough. If you have any questions or comments on any of my feedback, don't hesitate to reach out. Keep up the awesome work! 👍

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
 
-To see how well you are doing with your pipeline, take a look at
-the Win/Loss ratio.
+The *CRM* app manages the sales pipeline as leads/opportunities move from stage to stage,
+origination to sale (Won) or archival (Lost).
```
either are fine.
this feedback is optional.

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
-From there you can filter to which opportunities you wish to see, yours,
-the ones from your sales channel, your whole company, etc. You can then
-click on filter and check Won/Lost.
+To open the *Pipeline Analysis*, go to :menuselection:`CRM app --> Reporting --> Pipeline`.
```
The feedback here @MaxKeilOdoo is specificity of language.

We don't *open* the page, per se, we *access* it by navigating to it in Odoo.
It's a super nitpick but I agree with KC's suggestion here (nitpicks like this are also a sign you're almost done with the doc, fwiw 😉)

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
 
-.. image:: win_loss/win_loss02.png
-    :align: center
+Navigating the analysis page
+============================
```
agree.
also try to use simple present instead of continuous (navigate > navigating) whenever possible 👍

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+============================
 
-You also have the ability to switch to a pie chart view.
+Upon opening the :guilabel:`Pipeline Analysis` page, a graph of the opportunities (active leads)
+from the past year automatically populates. The bars represent the number of opportunities in each
+stage of the sales pipeline, color-coded to show the month the opportunity reached that stage.
```
I think the edit here was just specificity again. What type of graph? A *bar* graph.

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+  :guilabel:`My dashboard` in the *Dashboards* app, or linking the graph in a spreadsheet in the
+  *Documents* app. (See :ref:`Saving and sharing reports <win_loss/save_reports>` for more
+  information)
+- :guilabel:`Search bar`: Shows the filters and groupings currently being applied to the graph. To
+  add new filters/groups, type them into the search bar or click the :guilabel:`down arrow` at the
+  end of the bar to open a menu of options. (See :ref:`Search Options <win_loss/search>` for more
+  information)
```
Can still use the emoji ⬇️ I think.

Also depending on how it's rendered in Odoo, other variations of the downward arrow are ok like these more basic text symbols which are universally supported: ↓ or ▼, for example.

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,667 @@
+  add new filters/groups, type them into the search bar or click the :guilabel:`down arrow` at the
+  end of the bar to open a menu of options. (See :ref:`Search Options <win_loss/search>` for more
+  information)
+- :guilabel:`Graph`: Switches the analysis to Graph View. By default, this is selected.  (See
+  :ref:`View Options <win_loss/view>` for more information)
+- :guilabel:`Pivot`: Switches the analysis to Pivot View, which displays the results in a table.
+- :guilabel:`Cohort`: Switches the analysis to Cohort View, which displays the results by their
+  :guilabel:`Created on` week and :guilabel:`Closed Date` week.
+- :guilabel:`List`: Switches the analysis to List View, which displays the results in a single list.
```
Beginning with a navigational queue on the UI + lowercase first letter after colon ☝️

MaxKeilOdoo commented:
Hi @StraubCreative and @ksc-odoo , all requested changes have been made. This is ready for final review. Please let me know if you have any further suggestions.

StraubCreative commented:
Fixed commit message in [aaa5ef5](https://github.com/odoo/documentation/pull/6746/commits/aaa5ef5fc1ee8acdddab6b65ba6162d514ccade8).

@ksc-odoo I'll handle the next review for merge. Thank you for your detailed feedback and guidance so far 👍

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,702 @@
+   - **To create more complex rules:** click the :guilabel:`Add branch` icon to the right of the
+     rule. This adds another modifier below the rule for adding an "all of" or "any of" statement.
+
+.. image:: win_loss/custom-filter-add-branch.png
```
show a much more completed example to visually drive in the lesson and the value

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,702 @@
+   - **To remove a custom group:** click the :guilabel:`✖️ (x)` icon beside the custom group in the
+     search bar.
+
+.. image:: win_loss/add-custom-group.png
```
don't need

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,702 @@
+leads/opportunities that match the search criteria, but can be changed to measure other items of
+interest.
+
+.. image:: win_loss/measure-button-menu.png
```
don't need

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,702 @@
+To change the pipeline to a different view, click one of the four view icons, located in the
+top-right of the :guilabel:`Pipeline Analysis` page.
+
+.. image:: win_loss/pivot-and-list-views.png
```
don't need

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,702 @@
+Win/Loss reports
+----------------
+
+Win/Loss is a metric of the leads/opportunities marked "Won" or "Lost" over a period of time. By
+calculating "opportunities won" over "opportunities lost," teams can identify what methods and/or
+individuals are effectively converting leads into sales.
```
the punctuation in this section is unnecessary and distracting...
here's how I would simplify the punctuation and expand on the lesson that's trying to be communicated here.

```suggestion
Win/Loss is a calculation of active or previously active leads in a pipeline that were either marked
as **Won** or **Lost** over a specific period of time. By calculating opportunities won over
opportunities lost, teams can identify clarify key performance indicators (KPIs) that are converting
leads into sales, such as specific teams or team members, certain marketing mediums or campaigns,
and so on.
```

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,702 @@
+   Win/Loss Ratio = \frac{Opportunities Won}{Opportunities Lost}
+   \end{equation}
+
+A win/loss report filters the leads/opportunities from the past year, whether "Won" or "Lost," and
+groups the results by their stage in the pipeline. Creating this report requires a custom filter,
+and grouping the results by :guilabel:`Stage`.
```
again i don't like the quotes (" ") and slash (/) punctuation when it's not necessary.

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,702 @@
 
-.. image:: win_loss/win_loss03.png
-    :align: center
+.. image:: win_loss/pipeline-analysis-page.png
```
[screenshot is not in HD](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#screenshots), ideally the width caps to about 933px.

Double check the rest of the folder to make sure images are in HD and aren't too large in dimension. Remove any unnecessary screenshots or any that aren't used any more due to the rewrite.

StraubCreative reviewed (changes requested):
Hi @MaxKeilOdoo 
EXCELLENT doc! 🚀 
I have some suggestions for you below— 99% are optional.
The change request we'll need to do before merge is around screenshots, making sure that they're all in HD and dimensions don't expand beyond 933px. Once we minimally address that comment we should be good to push live 🙂 
Tag me again when we're ready, thanks!

MaxKeilOdoo commented:
@StraubCreative All requested changes made. Please let me know if this is fully approved. Thank you.

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,654 @@
+
+The :guilabel:`Pipeline Analysis` page can be customized with various filters and grouping options.
+
+To add new search criteria, type the desired criteria into the search bar, or click the :guilabel:`⬇️
+(down arrow)` icon, next to the search bar, to open a drop-down menu of all options. See the
+sections below for more information on what each option does.
```
```suggestion
To add new search criteria, type the desired criteria into the search bar, or click the
:guilabel:`⬇️ (down arrow)` icon, next to the search bar, to open a drop-down menu of all options.
See the sections below for more information on what each option does.
```

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,654 @@
+8. Once all rules have been added, click :guilabel:`Add` to add the custom filter to the search
+   criteria.
+
+   - **To remove a custom filter:** click the :guilabel:`✖️ (x)` icon beside the filter in the search
+     bar.
```
```suggestion
   - **To remove a custom filter:** click the :guilabel:`✖️ (x)` icon beside the filter in the
     search bar.
```

StraubCreative commented on content/applications/sales/crm/performance/win_loss.rst:
```diff
@@ -1,26 +1,654 @@
+Follow the steps below to create a win/loss report:
+
+1. Navigate to :menuselection:`CRM app --> Reporting --> Pipeline`.
+2. On the :guilabel:`Pipeline Analysis` page, click the :guilabel:`⬇️ (down arrow)` icon, next to the
+   search bar, to open a drop-down menu of filters and groupings.
```
```suggestion
2. On the :guilabel:`Pipeline Analysis` page, click the :guilabel:`⬇️ (down arrow)` icon, next to
   the search bar, to open a drop-down menu of filters and groupings.
```

StraubCreative reviewed (approved):
Hi @MaxKeilOdoo 
Great work 👏  Congrats on publishing your first doc!
I caught a couple lines that were too long but will fix in a following commit and merge in.
See you on the next PR!

StraubCreative commented:
@robodoo r+
